language/en-GB/en-GB.plg_vmpayment_xendit.ini

```diff
@@ -12,7 +12,7 @@ VMPAYMENT_XENDIT_CONF_PUBLIC_KEY_TIP="Unique Public API Key given by Xendit. Cas
 VMPAYMENT_XENDIT_CONF_SECRET_KEY_TIP="Unique Secret API Key given by Xendit. Case Sensitive!"
 VMPAYMENT_XENDIT_CONF_PAYMENT_TYPE_LABEL="Payment Type"
 
-VMPAYMENT_XENDIT_PARAMETER_REQUIRED="Xendit API Keys have not been configured."
-VMPAYMENT_XENDIT_UNSUPPORTED_CURRENCY="Xendit Payment Gateway does not support currency other than IDR (Indonesian Rupiah)."
+VMPAYMENT_XENDIT_PARAMETER_REQUIRED="Please include your API key as an Authorization header. Code: 100009"
+VMPAYMENT_XENDIT_UNSUPPORTED_CURRENCY="%s is currently not supported. Code: 100008"
 VMPAYMENT_XENDIT_INVALID_AMOUNT="Xendit Payment Gateway does not support decimal order amount."
```

Comment:
shopify also has this decimal amount error & I've asked yano to add a new generic error code for this in the sheet, later on you can check & see.